eq-author/src/App/page/Design/answers/BasicAnswer/index.js

```diff
@@ -62,9 +68,28 @@ export const StatelessBasicAnswer = ({
   optionErrorMsg,
   multipleAnswers,
 }) => {
-  let [toggled, setToggled] = useState(false);
+  const [toggled, setToggled] = useState(false);
+  const [mutuallyExclusiveLabel, setMutuallyExclusiveLabel] = useState();
```

Comment:
Would suggest not using state for this - this is stuff that should flow in from above when the parent re-renders post update (when Apollo result comes back) imo.

Toggle state would therefore be managed by whatever's set in the Apollo cache

Reply:
Hey Tom. Thanks for the comments! 

I've actioned the one regarding the toggle state; that was indeed a bit overkill. Although, the label text needs to have a `useState` otherwise the user can't update the input box. It uses a `useEffect` to populate the state whenever the `answer` changes, so it's initial value is being handeled by the cache.

Reply:
Oops, aye, that makes sense re: the label if you're using a controlled input.